worldengine/basic_map_operations.py

```diff
@@ -1,9 +1,12 @@
 import math
-import random
+import numpy
 
 
-def random_point(width, height):
-    return random.randrange(0, width), random.randrange(0, height)
+def random_point(width, height, randint_substitute=None):
```

Comment:
I could remove it, but then I would have to at least add a clarifying comment.^^ If random_point() were to be called from e.g. within temperature.py, I think it would be very important that it was *not* the global RNG that were to be queried. That would remove random numbers from the "stack" and thus influence other results depending on how often it would be called. It also would make the results for temperature.py a little non-deterministic.

Is it the parameter itself that's bugging you or rather the fact that the method now is four lines long instead of one? If it's both, I will change it back and add little note.